cerberus-web/src/main/java/com/nike/cerberus/util/AwsIamRoleArnParser.java

```diff
@@ -18,14 +18,25 @@
 
 import com.nike.backstopper.exception.ApiException;
 import com.nike.cerberus.domain.DomainConstants;
+import com.nike.cerberus.error.DefaultApiError;
 import com.nike.cerberus.error.InvalidIamRoleArnApiError;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 /** Utility class for concatenating and parsing AWS IAM role ARNs. */
 @Component
 public class AwsIamRoleArnParser {
+  private final boolean awsChinaEnabled;
+  private final boolean awsGlobalEnabled;
+
+  public AwsIamRoleArnParser(
+      @Value("${cerberus.partitions.awsGlobal.enabled}") boolean awsGlobalEnabled,
```

Comment:
Do we want both AWS Global and AWS China to be able to be enabled at the same time? Do we want to be more prescriptive and block that functionality for security reasons?

Reply:
I don't think it hurts to allow both since it may be a valid use case for some organizations.